example/lib/main.dart

```diff
@@ -100,6 +100,7 @@ class _HomeState extends State<Home> {
                   minHour: 9,
                   maxHour: 21,
                   is24HrFormat: false,
+                  displayBanner: false
```

Comment:
Please remove this

Reply:
Sorry for this fail, I forget to remove my test modification :(